identity-provider-service/README.md

```diff
@@ -265,3 +266,43 @@ being processed, then an error can be returned:
     "detail": "Identity object does not exist"
 }
 ```
+
+# Identity recovery
+The `state` field of the `GET` request to the endpoint `/api/v1/recover` should be urlencoded string
+containing valid JSON of the form
+```json
+{
+    "idRecoveryRequest": {
+        "v": 0,
+        "value": {
+            "idCredPub": "b05e024461f74ef98d14d6ae17121df28bc397fbfd55fedf08889c48c59d92e61c6853c561f054a9a1b6ad648d1da154",
+            "proof": "043b4af07edd7b5a694ac3034df622fd0184450ebf542bb49730cf38eba0b4464d51e0209c147edaaffaf9fc5fc1f2928aed95dc8c9ae15bf5497bc3255394f7",
+            "timestamp": 1657528974
+        }
+    }
+}
+```
+
+In case of success (if the request is valid), the response will be of the form
+```json
+{
+    "identityRetrievalUrl": "http://localhost:8100/api/v1/identity/ab4c3d1b3b4e183698dab33e8a0c7ba3e0eb72ebc0b6e916f55fda62509c9db"
+}
+```
+
+
+If the timestamp in the request is too far (given by the `--recovery-timestamp-delta` option, default is 60 seconds) from the current time, the response will be
+```json
+{
+    "code": 400,
+    "message": "Invalid timestamp."
+}
+```
+
+Otherwise, if the timestamp is OK, but the request is invalid due to an invalid proof, the response will be
+```json
+{
+    "code": 400,
+    "message": "Invalid ID recovery proof."
+}
```

Comment:
Can you specify what the HTTP status code will be as well? It should be 400 - Bad Request.